test/fastly-rails_test.rb

```diff
@@ -43,8 +43,8 @@
 
     it 'should have configuration options set up' do
       assert_equal api_key, configuration.api_key
-      assert_equal user, configuration.user
-      assert_equal password, configuration.password
+      assert_nil user, configuration.user
```

Comment:
this doesn't quite do what the original intent was. `assert_nil`'s second argurment is an optional message for when the assertion fails.

we should change this to be `assert_nil configuration.user` to keep the original intent of the test and avoid the minitest deprecation warning.

Reply:
Good catch! Unintentional and overlooked on my part. Reverted.